source/botbotbot.py

```diff
@@ -119,6 +119,9 @@ def recv_message(self, content='', parent=None, this_message=None, sender='', se
                 self.pause(pause_text='/me has been paused by "' + sender + '".', reply_to=this_message)
         elif self.paused and EuphUtils.command('!help', self.nickname).match(content):
             self.pause(reply_to=this_message)
+        elif content == '!antighost':
```

Comment:
I looked into that, however it appeared that the EuphUtils.command() method checks to see if the @mention is in the command, so I decided to do it this way.  If you want I could write a method in EuphUtils that doesn't require the @mention to be present but does all the other checking.  If so let me know what the method should be called.